site/content/en/docs/Guides/fleet-updates.md

```diff
@@ -97,16 +97,15 @@ the `v1` Fleet, like so:
 apiVersion: "allocation.agones.dev/v1"
 kind: GameServerAllocation
 spec:
-  required:
-    matchLabels:
-      game: my-awesome-game
-  preferred:
+  selectors:
```

Comment:
Changes to the website shouldn't be published until the next release. You need to use the `{{% feature publishVersion="1.17.0" %}}` blocks for any changes to the website.

https://agones.dev/site/docs/contribute/#within-a-page

Reply:
Done.